pages/common/gifsicle.md

```diff
@@ -1,20 +1,32 @@
 # gifsicle
 
-> Create GIFs.
+> GIF manipulator.
 > More information: <https://www.lcdf.org/gifsicle>.
 
-- Optimise a GIF:
+- Optimize a GIF as a new file:
 
-`gifsicle --batch --optimize=3 {{amin.gif}}`
+`gifsicle {{path/to/input.gif}} --optimize=3 -o {{path/to/output.gif}}`
 
-- Make a GIF animation with gifsicle:
+- Unoptimize a GIF in place:
 
-`gifsicle --delay={{10}} --loop *.gif > {{anim.gif}}`
+`gifsicle -b {{path/to/input.gif}} --unoptimize`
 
-- Extract frames from an animation:
+- Extract a frame from an animation:
 
-`gifsicle {{anim.gif}} '#0' > {{firstframe.gif}}`
+`gifsicle {{path/to/input.gif}} '#{{0}}' > {{path/to/firstframe.gif}}`
 
-- You can also edit animations by replacing, deleting, or inserting frames:
+- Make a GIF animation from selected still/animated GIFs:
 
-`gifsicle -b {{anim.gif}} --replace '#0' {{new.gif}}`
+`gifsicle {{*.gif}} --delay={{10}} --loop > {{path/to/output.gif}}`
+
+- Reduce file size using lossy compression:
+
+`gifsicle -b {{path/to/input.gif}} --optimize=3 --lossy={{100}} --colors={{16}} --dither`
+
+- Delete the first 10 frames and all frames after #20 from an animation:
+
+`gifsicle -b {{path/to/input.gif}} --delete '#{{0-9}}' '#{{20-}}'`
+
+- Modify an animation with various transformations. Put these before the animation filename to avoid specifying frame ranges:
```

Comment:
```suggestion
- Modify an animation with various transformations:
```

Again, although useful, it doesn't meet the style of pages.

Reply:
I wish there was reason to include it; the rest of the examples place the flags after the filename, but this is the first place ordering is important. I was kinda playing off the style of `ffmpeg`'s page here.

Reply:
What about this? It's a more subtle implication.

```suggestion
- Modify all frames in a GIF with transformation options:
```

Reply:
Its better, lets see what others think @navarroaxel @kbdharun

Reply:
Perhaps
```suggestion
- Modify all frames from a GIF file using specific transformation options:
```

Reply:
I like that, thank you.

Reply:
```suggestion
- Modify all frames of a GIF using specific transformation options:
```

Reply:
I think you can go ahead and commit your suggestion @CairnThePerson

Reply:
Ah, thanks for the ping.